docs/protocol.md

```diff
@@ -204,6 +209,81 @@ Here are the different methods that can be called:
 * Heartbeat
 * GetMessagesById
 
+### Greeting a server
+🧭 **RPC Message** > **Query** > **Greet Server**
+
+By sending a greet server message, a server can transmit its public key as well as the 
+addresses of its client and server endpoints to another server.
+
+This message should be sent by the server that opens a connection with another server. It should trigger
+a `GreetServer` message from the server that receives the connection containing its own information. 
+
+RPC
+
+```json5
+// ../protocol/examples/query/greet_server/greet_server.json
+
+{
+    "jsonrpc": "2.0",
+    "method": "greet_server",
+    "params": {
+        "public_key": "J9fBzJV70Jk5c-i3277Uq4CmeL4t53WDfUghaK0HpeM=",
```

Comment:
I see that the sender field is in messages that are related to the LAO. However, there might be a few cases where we would need the public key of the other servers before a LAO is created and after the moment a connection is established. One example from the discussions during the decentralized communications meeting was (in the case where two servers open a connection which each other instead of only one) deciding which connection to drop could depend on the public key. What do you think ?

Reply:
They're not sent on any channel so I think in that sense they're not using the pub/sub model and, if I understood the documentation properly, they shouldn't contain that field. We have other messages that operate in the same way (heartbeats and getMessagesById which I mentionned in another comment).

Reply:
I see, that explains my confusion!
Yes I guess then my comments don‘t make sense / you must include the public key.

But I did not realize these messages are completely different from all the others. I would reccomend putting them in a completely separate section in the documentation to make this clear. (including the heartbeat and getMessagesById)

Reply:
Sorry I was confused, I think I now unterstand!

Reply:
Since there is now payload, there won‘t be any mid-level protocol of course. Sorry, not sure what my mind was doing.